flytekit/configuration/internal.py

```diff
@@ -66,6 +66,11 @@ class AZURE(object):
     CLIENT_SECRET = ConfigEntry(LegacyConfigEntry(SECTION, "client_secret"))
 
 
+class LOCAL(object):
```

Comment:
Local

Reply:
Thanks, fixed in https://github.com/flyteorg/flytekit/pull/2106/commits/836398ab5382056bd7436857f48b4e63ffe2af9a